### PR TITLE
Fix thumbnail submission request not working after migration to https

### PR DESCRIPTION
**Pull request by kprist**

### CODE REVIEW

File: application/controllers/handler.js [hunk 1]
```diff
@@ -1387,47 +1387,19 @@ let self = module.exports = {
 };
 
 function createThumbnail(slideContent, slideId, user) {
-    let http = require('http');
+    let rp = require('request-promise-native');
```
Comment (bwulff) [inline application/controllers/handler.js]: Not sure about the lazy initialization of packages... 
Dejan found a post about the topic: http://stackoverflow.com/a/9139419

Comment (bwulff) [inline application/controllers/handler.js]: Ali says: "yes, indeed it is helpful. dynamic importing, lazy loading, and code splitting/bundling are some topics we need to discuss for optimization"

Comment (kprist) [inline application/controllers/handler.js]: Not sure what you're saying here, is this something that you want me to change before merging?

Comment (bwulff) [inline application/controllers/handler.js]: It was new to me to see the require(..) at the beginning of a function and I was wandering if this a good thing to do. But it seems to be fine and according to Ali even a kind of optimization (--> the library loaded the first time when the functions is executed == lazy loading). So it's perfectly fine.

Comment (kprist) [inline application/controllers/handler.js]: Oh OK, thought it was an issue for merging, anyway, should I leave you to merge it at your convenience? or should I? I thought this was urgent

**Pull request merged**